controllers/tipo_retencion.go

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import (
+	"github.com/astaxie/beego"
+	"github.com/udistrital/cuentas_contables_crud/helpers"
+	"github.com/udistrital/cuentas_contables_crud/managers"
+	"github.com/udistrital/cuentas_contables_crud/models"
+	"go.mongodb.org/mongo-driver/mongo"
```

Comment:
(En general - comentario menor) Tratar de dejar los import separados (agrupados?) según:
1. paquetes core, oficiales de Go ("net", "fmt", "url", entre otros)
2. paquetes fuera de github/udistrital
3. paquetes github/udistrital

Algo así (se puede aplicar este _suggested change_ via web)
```suggestion
	"github.com/astaxie/beego"
	"go.mongodb.org/mongo-driver/mongo"

	"github.com/udistrital/cuentas_contables_crud/helpers"
	"github.com/udistrital/cuentas_contables_crud/managers"
	"github.com/udistrital/cuentas_contables_crud/models"
```